src/cmd/sanity_test/common/common.go

```diff
@@ -19,33 +19,40 @@ type PermissionInfo struct {
 }
 
 const (
-	sanityBackupID             = "SANITY_BACKUP_ID"
-	sanityTestSourceContainer  = "SANITY_TEST_SOURCE_CONTAINER"
-	sanityTestRestoreContainer = "SANITY_TEST_RESTORE_CONTAINER"
-	sanityTestUser             = "SANITY_TEST_USER"
+	sanityBackupID                   = "SANITY_BACKUP_ID"
+	sanityTestSourceContainer        = "SANITY_TEST_SOURCE_CONTAINER"
+	sanityTestRestoreContainer       = "SANITY_TEST_RESTORE_CONTAINER"
+	sanityTestRestoreContainerPrefix = "SANITY_TEST_RESTORE_CONTAINER_PREFIX"
+	sanityTestUser                   = "SANITY_TEST_USER"
+	sanityTestCategory               = "SANITY_TEST_CAT"
```

Comment:
Nit: no need to shorten the word.
```suggestion
	sanityTestCategory               = "SANITY_TEST_CATEGORY"
```